src/libsyntax/print/pprust.rs

```diff
@@ -708,6 +708,22 @@ pub trait PrintState<'a> {
         Ok(())
     }
 
+    fn print_attribute_path(&mut self, path: &ast::Path) -> io::Result<()> {
```

Comment:
Looks like existing `print_path` can be used instead of introducing this new function.

Reply:
`print_path` is not a method of `PrintState` trait, so I am not sure how to exploit it. Should I write like `self.writer().word(&path_to_string(path))`?